manifests/kadalu-operator.yaml

```diff
@@ -191,8 +191,8 @@ spec:
           securityContext:
             capabilities: {}
             privileged: true
-          image: docker.io/kadalu/kadalu-operator:latest
-          imagePullPolicy: Always
+          image: docker.io/kadalu/kadalu-operator:test
```

Comment:
Is this change only for testing?

Reply:
Yes it was for testing. Removed the file.